magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationDelegate.kt

```diff
@@ -151,7 +151,9 @@ class NavigationDelegate(
     callOnNavigate(currentNavigable!!)
     when (currentState) {
       is LifecycleState.Shown, is LifecycleState.Resumed -> {
-        containerView!!.addView(currentNavigable!!.view!!)
+        containerView!!.addView(
+          currentNavigable!!.view!!,
+          direction.indexToAddView(containerView!!))
```

Comment:
This was a bug which caused a random glitch while i was testing the transition. Fixed now.